src/app/app.component.html

```diff
@@ -1,10 +1,16 @@
 <h1>
   Components App
 </h1>
+
+
 <app-pizza-toppings></app-pizza-toppings>
 <br>
 <app-tjs-multiplication></app-tjs-multiplication>
 <br>
+<hr><hr>
```

Comment:
Could you put your component at the bottom? Simply make the change locally, verify it's working, then push to update your PR branch. Thanks.

Reply:
I went ahead and did this for you so that I could get another student lined up for their PR.